src/test/java/org/example/action/security/cookiesession/DoLoginAction.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024-2024, Inversoft Inc., All Rights Reserved
```

Comment:
`2024`, looks like this is in a lot of places.

Reply:
Fixed 566308936463322304bf51a2b627e800480877dd